Gemfile.lock

```diff
@@ -348,3 +346,6 @@ DEPENDENCIES
   watir-webdriver
   web-console (~> 2.0)
   whenever
+
+BUNDLED WITH
```

Comment:
I'm not sure what the proper way to do this is, so I just deleted the .lock file and re-ran bundle install, and this is what resulted.